libvast/src/format/zeek.cpp

```diff
@@ -276,13 +266,21 @@ caf::error reader::read_impl(size_t max_events, size_t max_slice_size,
   // Sanity checks.
   VAST_ASSERT(max_events > 0);
   VAST_ASSERT(max_slice_size > 0);
+  auto next_line = [&] {
+    auto timed_out = lines_->next_timeout(read_timeout_);
+    if (timed_out)
+      VAST_DEBUG(this, "reached input timeout at line", lines_->line_number());
+    return timed_out;
+  };
```

Comment:
Should this be a member function on the line range instead to deduplicate it?

Reply:
The lambda only exists so the log statement is not duplicated. If we move it to the line range the message would read `vast.detail.line_range reached ...` instead of `vast.format.zeek.reader reached ...`.

Reply:
Fair point.